viite-UI/src/view/NodeSearchForm.js

```diff
@@ -112,7 +112,10 @@
     };
 
     var nodePointTemplatesHtml = function (nodePointTemplates) {
-      var groups = _.groupBy(nodePointTemplates, function (template) {
+      var uniqueNPTemplates = _.uniqWith(nodePointTemplates, function (o1, o2) {
```

Comment:
I would write this as something like:
`_.uniqBy(nodePointTemplates, ['roadNumber', 'roadPartNumber', 'addrM'])`

Reply:
This gives only one item in list so, reverted and using original uniqWith instead.